hack/elk-flow-collector/deploy.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
I suggest trying to get rid of this script as it seems like it should be possible to me. Can we investigate the following:
- use the K8s Pod downward API (https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#capabilities-of-the-downward-api) to set an environment variable with the public IP of the Pod
- use an environment variable in the logstash configuration
```
input {
  udp {
    host => "${POD_IP}"
...
```
- same thing in logstash.yml

Reply:
Thanks for your idea, this approach works well.